docker-compose.yaml

```
@@ -92,39 +93,43 @@ services:
+      platforms:
+        - linux/amd64
     environment:
-      FIREBASE_PROJECT_ID: <your_firebase_project_id>
-      FIREBASE_PRIVATE_KEY: <your_private_key>
-      FIREBASE_CLIENT_EMAIL: <your_client_email>
-      KAFKA_BROKER: kafka:9092
-      KAFKA_CLIENT_ID: notification-consumer
-      CONSUMER_TYPE: notification
-    restart: always
+      #Sms broker environment variables
+      TWILIO_ACCOUNT_SID : <your_twilio_accound_sid> 
```

yamllint (1.35.1)

[warning] too many spaces before colon

(colons)